universal-login-wallet/test/integration/helpers/setupUI.tsx

```diff
@@ -13,7 +13,7 @@ export const setupUI = async (relayer: Relayer, tokenAddress?: string) => {
   const [wallet] = await getWallets(relayer.provider);
   const tokens = tokenAddress ? [tokenAddress, ETHER_NATIVE_TOKEN.address] : [ETHER_NATIVE_TOKEN.address];
   const services = await createPreconfiguredServices(relayer.provider, relayer, tokens);
-  services.tokenService.start();
```

Comment:
Duplication. We should extract `start` to `Services` class.

Reply:
This is test tho